crates/rpc/src/jsonrpc/router.rs

```diff
@@ -117,7 +117,9 @@ pub async fn rpc_handler(
 ) -> impl axum::response::IntoResponse {
     // Only json content allowed.
     if content_type != ContentType::json() {
-        return StatusCode::UNSUPPORTED_MEDIA_TYPE.into_response();
+        if &content_type.to_string() != "application/json; charset=utf-8" {
```

Comment:
This should maybe account for whitespaces? Would complicate matters a fair bit though.

Reply:
Solved by switching to `mime` crate